e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/about-unissued-facility.spec.js

```diff
@@ -0,0 +1,234 @@
+import { format } from 'date-fns';
+
+import relative from '../../../relativeURL';
+
+import CONSTANTS from '../../../../fixtures/constants';
+
+import dateConstants from '../../../../../../e2e-fixtures/dateConstants';
+
```

Comment:
Can we remove these empty lines inbetween imports for E2E tests in this PR please

Reply:
Actually, going to resolve this as i've made the same changes in [this PR](https://github.com/UK-Export-Finance/dtfs2/pull/3556)

Will address in there